src/acme/ui/AcmeUI.java

```diff
@@ -0,0 +1,151 @@
+package acme.ui;
+
+import acme.pd.*;
+
+import javax.swing.*;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowListener;
+
+public class AcmeUI extends JFrame {
+
+    private Company company;
+
+    public AcmeUI() {
+        super("Acme Delivery Software");
+
+        this.company = new Company();
+        this.buildMenu();
+
+        setVisible(true);
+        setSize(550, 550);
+        setLocationRelativeTo(null);
+        addWindowListener(new ShutdownListener());
+
+        loginScreen();
+    }
+
+    private void buildMenu() {
+        JMenuBar menuBar = new JMenuBar();
+
+        JMenu ticketMenu = new JMenu("Tickets");
+        menuBar.add(ticketMenu);
+
+        // TODO actually implement.  This is temporary to get a base for everyone
+        ticketMenu.add(new JMenuItem("Create"));
+        ticketMenu.add(new JMenuItem("List"));
+
+        this.setJMenuBar(menuBar);
+    }
+
+    public void setPanel(AcmeJPanelBase panel) {
+        this.getContentPane().removeAll();
+        this.getContentPane().add(panel);
+        panel.buildPanel();
+        this.revalidate();
+        this.repaint();
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void loginScreen() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void ticketList() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
```

Comment:
Could you create an empty `TicketListJPanel` that navigates the user to (what can on this branch be) an empty screen? Just so everyone has a clear example of how the `JMenuItem` ties to this function you've created.

Reply:
I would prefer just writing some documentation instead of creating the ticketlistjpanel.  Simply because I have the examplejpanel which covers pretty much everything someone would want to do